apps/prairielearn/src/server.js

```diff
@@ -594,7 +594,7 @@ module.exports.initExpress = function () {
       res.locals.navPage = 'request_course';
```

Comment:
I'd also be OK refactoring setting `res.localsNavPage` inside the page so that we can save a few lines in server.js but maybe there's some other reason we're setting those nav variables in server.js.

Reply:
Done in b5c116a